### PR TITLE
Added Japanese support for getting string length.

### PR DESCRIPTION
**Pull request by fuwasegu**
# TL;DR
Fixed using` mb_strlen()` to get the string length instead of strlen().

# Details
* For elements such as placeholder and tittle, where the number of characters that can be entered is limited, the specification is to get the number of characters and throw an exception if it exceeds the limit.
* The `strlen()` method is used to get the length of a string.
* For strings consisting of only alphabetic characters (e.g. ASCII encoding), the strlen() method will give correct results, but for strings containing Japanese or other characters (e.g. UTF-8 encoding), it may not give correct results.
* This is because the `strlen()` method is designed to return the number of bytes, not the number of characters, and the number of characters and the number of bytes match for encodings with 1 byte per character, such as alphabetic characters, but not for encodings with around 3 bytes per character, such as Japanese (UTF-8).
* In this fix, `mb_strlen()`, which supports multibyte characters, is used so that even Japanese string lengths can be obtained correctly.

Thanx.

### CODE REVIEW

**Review by makies — APPROVED** [review]

**Review by jeremeamia — APPROVED** [review]

**Comment by jeremeamia** [comment]
Thank you. This is a good change.
